### PR TITLE
povray: update to 3.7.0.8

### PR DESCRIPTION
**Pull request by luzpaz**
Testing against the CI. Looks like upstream fixed some issues and patches may be  superfluous

### CODE REVIEW

**Comment by pmetzger** [comment]
@luzpaz Any updates?

**Comment by luzpaz** [comment]
Sorry, AFK because of the holidays. Will not get to this till a week or so

File: graphics/povray/Portfile [hunk 1]
```diff
@@ -4,8 +4,8 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           muniversal 1.0
 
-github.setup        POV-Ray povray 3.7.0.0 v
-revision            6
+github.setup        POV-Ray povray 3.7.0.8 v
+revision            0
```
[the remark this answers is not in the document]
Comment (pmetzger) [reply graphics/povray/Portfile]: BTW, this has now been changed, revision lines are now retained, so ignore my comment.

**Comment by pmetzger** [comment]
Okay, I'm going to close this for now. @luzpaz, when you get a chance to work on it, we can re-open this PR or you can start a new one.

**Comment by ryandesign** [comment]
In any case, thanks for attempting the update and for notifying us it was available; I had missed it. I'll work on it now.

**Pull request closed without merging**